### PR TITLE
Google K8s Marketplace

### PR DESCRIPTION
**Pull request by eranchetz**

### CODE REVIEW

**Comment by ushachar** [comment]
@eranchetz - This repo is used for thing other then just GKE marketplace...

File: README.md [hunk 1]
```diff
@@ -1,220 +1,300 @@
-### Deploying Redis Enterprise K8s using an operator (custom controller)
-### Table of Contents
+# Overview
```
Comment (ushachar) [inline README.md]: Why not just create a GKE-MARKETPLACE-README.md?
Is there a specification on repo structure?

Comment (eranchetz) [reply README.md]: @ushachar my goal was to create a new RedisLabs:GKE_Marketplace branch, if it doesn't make sense, maybe we can have a new repo for that?
cc: @amiramm

**Comment by eranchetz** [comment]
I will put everything in new dir

**Pull request closed without merging**